src/aws-cpp-sdk-core/source/http/windows/WinHttpSyncHttpClient.cpp

```diff
@@ -538,8 +538,11 @@ void* WinHttpSyncHttpClient::OpenRequest(const std::shared_ptr<HttpRequest>& req
 {
     LPCWSTR accept[2] = { nullptr, nullptr };
 
-    DWORD requestFlags = WINHTTP_FLAG_REFRESH |
-        (request->GetUri().GetScheme() == Scheme::HTTPS && m_verifySSL ? WINHTTP_FLAG_SECURE : 0);
+    DWORD requestFlags = request->GetUri().GetScheme() == Scheme::HTTPS && m_verifySSL ? WINHTTP_FLAG_SECURE : 0;
+    if (m_usingProxy) {
```

Comment:
I've tried to remove this header without changing this option in the SDK (to preserve other WinHTTP request behavior on this option enabled). However, WinHTTP refused to remove it.